libs/features/mas/commerce/stats.json

```diff
@@ -0,0 +1 @@
ules/@pandora/data-source-wcs/esm/webCommerceArtifact.js":{"bytesInOutput":43},"../node_modules/@pandora/data-source-wcs/esm/index.js":{"bytesInOutput":0},"../node_modules/@dexter/tacocat-core/src/utilities.js":{"bytesInOutput":1338},"../node_modules/@dexter/tacocat-core/src/log.js":{"bytesInOutput":577},"../node_modules/@dexter/tacocat-core/src/promotion.js":{"bytesInOutput":383},"../node_modules/tslib/tslib.es6.mjs":{"bytesInOutput":870},"../node_modules/@formatjs/icu-messageformat-parser/lib/error.js":{"bytesInOutput":1683},"../node_modules/@formatjs/icu-messageformat-parser/lib/types.js":{"bytesInOutput":805},"../node_modules/@formatjs/icu-messageformat-parser/lib/regex.generated.js":{"bytesInOutput":54},"../node_modules/@formatjs/icu-skeleton-parser/lib/date-time.js":{"bytesInOutput":2124},"../node_modules/@formatjs/icu-skeleton-parser/lib/index.js":{"bytesInOutput":0},"../node_modules/@formatjs/icu-skeleton-parser/lib/regex.generated.js":{"bytesInOutput":47},"../node_modules/@formatjs/icu-skeleton-parser/lib/number.js":{"bytesInOutput":4200},"../node_modules/@formatjs/icu-messageformat-parser/lib/parser.js":{"bytesInOutput":16430},"../node_modules/@formatjs/icu-messageformat-parser/lib/index.js":{"bytesInOutput":517},"../node_modules/@formatjs/fast-memoize/lib/index.js":{"bytesInOutput":996},"../node_modules/intl-messageformat/lib/src/error.js":{"bytesInOutput":869},"../node_modules/intl-messageformat/lib/src/formatters.js":{"bytesInOutput":2230},"../node_modules/intl-messageformat/lib/src/core.js":{"bytesInOutput":2930},"../node_modules/intl-messageformat/lib/index.js":{"bytesInOutput":10},"../node_modules/@dexter/tacocat-consonant-templates/src/price/numberFormat.js":{"bytesInOutput":1700},"../node_modules/@dexter/tacocat-consonant-templates/src/price/utilities.js":{"bytesInOutput":2014},"../node_modules/@dexter/tacocat-consonant-templates/src/price/template.js":{"bytesInOutput":4163},"../node_modules/@dexter/tacocat-consonant-templates/src/price/index.js":{"bytesInOutput":106},"../node_modules/@dexter/tacocat-consonant-templates/src/discount/template.js":{"bytesInOutput":226},"../node_modules/@dexter/tacocat-consonant-templates/src/discount/index.js":{"bytesInOutput":12},"../node_modules/@dexter/tacocat-wcs-client/src/wcsUtilities.js":{"bytesInOutput":1032},"src/external.js":{"bytesInOutput":120},"src/constants.js":{"bytesInOutput":991},"src/lana.js":{"bytesInOutput":1116},"src/defaults.js":{"bytesInOutput":444},"src/utilities.js":{"bytesInOutput":687},"src/settings.js":{"bytesInOutput":2691},"src/log.js":{"bytesInOutput":1211},"src/placeholder.js":{"bytesInOutput":3171},"src/checkout-link.js":{"bytesInOutput":2938},"src/inline-price.js":{"bytesInOutput":2807},"src/checkout.js":{"bytesInOutput":1889},"src/ims.js":{"bytesInOutput":618},"src/literals.js":{"bytesInOutput":348},"src/price.js":{"bytesInOutput":1223},"src/wcs.js":{"bytesInOutput":1952},"src/service.js":{"bytesInOutput":1213},"src/index.js":{"bytesInOutput":0}},"bytes":79845}}}
```

Comment:
hihi this stats.json always finds a way back into the commits :D
i thought someone added it to gitignore already but seems like no

Reply:
I thought I ignored it, will do it.